ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -1427,6 +1472,19 @@ router bgp {{ router_bgp.as }}
 {%                 endif %}
 {%             endif %}
 {%         endif %}
+{%         if vrf.default_route_exports is arista.avd.defined %}
+{%             for default_route_export in vrf.default_route_exports %}
+{%                 set vrf_default_route_export_cli = "default-route export " ~ default_route_export.address_family %}
+{%                 if default_route_export.rcf is arista.avd.defined %}
+{%                     set vrf_default_route_export_cli = vrf_default_route_export_cli ~ " rcf " ~ default_route_export.rcf %}
+{%                 elif default_route_export.route_map is arista.avd.defined %}
+{%                     set vrf_default_route_export_cli = vrf_default_route_export_cli ~ " route-map " ~ default_route_export.route_map %}
+{%                 elif default_route_export.always is arista.avd.defined(true) %}
```

Comment:
always keyword should not be check in elif condition as it can come with rcf or route map
```
router bgp 65000
   vrf mahesh
      default-route export evpn
      default-route export vpn-ipv4 always route-map test
```

Reply:
Thanks for the catch. Done